### ISSUE TITLE
Option to display either username or display name in messages

### ISSUE THREAD

**Issue by ghost** [EssentialsX/Essentials#1406]
Hello,

Can u create in the new essentialsX update that not will been used the displayname for ' "displayname" has teleported to you'.

I use a prefix before the playername and that will be show before the playername in that messages.

For example: '[rookie] aeadnerr has teleported to you'

Can u set that to the playername and not to the displayname?

Best regards
Thibaud

**Comment by LaxWasHere** [comment]
All messages can be edited in messages.properties https://github.com/drtshock/Essentials/blob/2.x/Essentials/src/messages.properties

**Comment by mdcfe** [comment]
I don't think that it would be particularly easy to implement any way of supporting both simultaneously, since the messages currently substitute whatever string is passed to the tl method. Since in most cases the display name is passed, we'd need to add a condition to every use of translated messages which substitutes in player names, which is quite a lot of places.

**Comment by mdcfe** [comment]
Slight necro, but a realistic implementation of this could be to have a `User#getMessagesName` method which returns either the username or displayname of the user as set in the config to be used in `tl` calls.

**Comment by JRoy** [comment]
Closing as I do not feel this is niche enough to warrant the overhaul it commands.

You may achieve this already by disabling the following option in the config;

```yml
# Disable this if you have any other plugin, that modifies the displayname of a user.
change-displayname: true
```
